scripts/zkai_d128_gate_value_activation_fused_gate.py

```
@@ -0,0 +1,556 @@
+    for field, expected in checks:
+        if fused_input.get(field) != expected:
+            raise FusedGateError(f"fused handoff field drift: {field}")
+    if activation_input.get("source_gate_value_projection_output_commitment") != gate_input.get("gate_value_projection_output_commitment"):
+        raise FusedGateError("activation source output does not match gate/value output")
+    if activation_input.get("gate_projection_q8") != gate_input.get("gate_projection_q8"):
+        raise FusedGateError("activation gate vector does not match gate/value output")
+    if activation_input.get("value_projection_q8") != gate_input.get("value_projection_q8"):
+        raise FusedGateError("activation value vector does not match gate/value output")
+
+
+def validate_payload(payload: dict[str, Any]) -> None:
```

Ruff (0.15.12)

[warning] Too many branches (15 > 12)

(PLR0912)